airflow/macros/__init__.py

```diff
@@ -19,12 +19,15 @@
 import uuid  # noqa
 from datetime import datetime, timedelta
 from random import random  # noqa
-from typing import Any, Optional
+from typing import TYPE_CHECKING, Any, Optional
 
 import dateutil  # noqa
 
 from airflow.macros import hive  # noqa
 
+if TYPE_CHECKING:
```

Comment:
Why type checking ? Should not we just import it as usual? i think it has no performance penalty (DateTime will be imported by the time it reaches here).

Reply:
Updated